SourceEditor/build.gradle

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ */
+
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    apply from: 'dependencies.gradle'
+
+    repositories {
+        google()
+        jcenter()
```

Comment:
You can check by removing this line (repo) if there is any issue building the app. If there isn't any, then you can remove it. 
FYI Jcenter service has been shut down, right now you just can download libraries from there (so your code works) but we don't know till when the service will be available, so it's good that all our codebases get rid of jcenter asap.
(Google also recommends that).

Reply:
removing `jcenter()` does lead to build errors, but replacing it with `mavenCentral()` seems to fix the issue. Is this a valid alternative?

Reply:
Definitely! in fact, is what you should do. Good point!